tox.ini

```diff
@@ -19,4 +19,4 @@ commands =
 
 [testenv:upgrade]
 deps = pip-tools
-commands = pip-compile --upgrade --no-header --no-emit-options --output-file requirements.txt requirements.in
+commands = pip-compile --upgrade --no-header --no-emit-options --strip-extras --output-file requirements.txt requirements.in
```

Comment:
@akaIDIOT this shouldn't be a problem, right? 😅

Reply:
Not 100% sure what that flag does, but if the set of deps is what you wanted to have, I'd expect it to be fine. Don't think the sdk will need any of hansken.py's extras.